index.html

```diff
@@ -173,23 +173,23 @@ <h2 class="project-title">Multi-Post
 
     <section id="about-section">
       <div class="about-intro">
-        <h2 class="about-heading">About Myself</h4>
+        <h2 class="about-heading">About Myself</h2>
 
         <p class="about-description">Hello I'm a software developer! I can help you build a product, feature or website.
           Look through some of my work and experience. If you like what you see and have a project you need coded, don't
           hestiate to contact me.'
         </p>
 
         <div class="social-media">
-          <h2 class="connect">LET'S CONNECT</h2>
+          <h3 class="connect">LET'S CONNECT</h3>
 
           <div class="social-links2">
       <ul>
         <li><a href="https://twitter.com/YoungeCjay" target="_blank">
           <img src="./images/icons/Twitter icon.png"></a></li>
         <li><a href="https://www.linkedin.com/in/clinton-younge-83386a25a/" target="_blank">
-          <img src="./images/icons/Linkedin icon.png"></li>
-        <li><a><img src="./images/icons/Letter M.png"></a></li>
+          <img src="./images/icons/Linkedin icon.png"></a></li>
+        <li><a href="#"><img src="./images/icons/Letter M.png"></a></li>
         <li><a href="https://github.com/ClintonYounge" target="_blank">
           <img src="./images/icons/Github Kitty.png"></a></li>
         <li><a href="#"><img src="./images/icons/Finger icon.png"></a></li>
```

Comment:
- [ ] Kindly add a working link to your `a` tag on lines 192 and 195 as this will help to improve your accessibility score and enhance user experience. Dead links are really not good mistakes from assumed professionals. In case you do not have a profile on the platform yet, I suggest you open new one, thankfully, it's free. What do you say? 😉 😄

Reply:
@AbayomiOlaoye Not sure as to which platforms those two icons represent. Mind informing me? I've been asking around to no avail.